### DISCUSSION TITLE
ESP32-2432S028 Wifi

### DISCUSSION THREAD

**Discussion by stream74** [HASwitchPlate/openHASP#757] [Hardware]
![Uploading Capture d'écran 2024-06-29 164703.png…]()
Hi, I'm trying to get openHASP working with an ESP32-2432S028

I first installed the Nightly Development Build 0.70-rc12, but I can't connect to the wifi. I tried developing locally with PlatformIO but the same problem occurs. I can't connect to the network, while with Esphome everything works perfectly.

**Comment by fvanroie** [comment]
What's the length of the SSID and password?

**Reply by stream74** [reply]
ssid 17
pass 13
but when i try with d1-mini-esp32 wifi works
